vms/event/forms.py

```diff
@@ -30,3 +30,17 @@ def clean(self):
 class EventDateForm(forms.Form):
     start_date = forms.DateField(required=False)
     end_date = forms.DateField(required=False)
+
+
+class SearchEventForm(forms.Form):
+    name = forms.RegexField(
+      regex=r'^[(A-Z)|(a-z)|(0-9)|(\s)|(\.)|(,)|(\-)|(!)|(\')]+$', max_length=75, required=False)
```

Comment:
Are all these validations in accordance with the event fields in the rest of code?